lucet-spectest/src/lib.rs

```diff
@@ -112,10 +124,11 @@ fn step(script: &mut ScriptEnv, cmd: &CommandKind) -> Result<(), SpecTestError>
                 ref field,
                 ref args,
             } => {
+                println!("invoke {:?} {} {:?}", module, field, args);
```

Comment:
is the `println` here (and elsewhere in this file) intentionally still present? I noticed it when running spectests on my branch but hadn't looked at where it came from. If it is intentional, what for?

Reply:
They're pretty ad-hoc and I'd like to find a better solution by refactoring lucet-spectest a bit, but for now they're quite useful during debugging if a spec test causes a segfault, which did happen occasionally.